### DISCUSSION TITLE
Contributor

### DISCUSSION THREAD

**Discussion by PR33T0** [blazority/support#127] [General]
Hi, 

Are you looking for contributors?

**Comment by jens-cailliau** [comment]
@karan-kang Or are you willing to make the code open-source?

**Reply by jens-cailliau** [reply]
@karan-kang : any feedback?

**Comment by karan-kang** [comment]
At the moment, we have chosen to not make the library open source, but rather free to use with support.
Thanks for interest, if there is any specific use-case that you would like us to support, feel free to log a issue.
